core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -270,9 +270,25 @@ private[spark] object JettyUtils extends Logging {
 
       gzipHandlers.foreach(collection.addHandler)
       connectors.foreach(_.setHost(hostName))
+      // As each acceptor and each selector will use one thread, the number of threads should at
+      // least be the number of acceptors and selectors plus 1. (See SPARK-13776)
+      var minThreads = 1
+      connectors.foreach { c =>
```

Comment:
I want to crash the app if we add a non-SelectChannelConnector in future. It's better than being silent.